operator/pkg/cli/upgrade.go

```diff
@@ -74,6 +75,10 @@ func UpgradeCmd() *cobra.Command {
 
 	// TODO(upgrade): local-artifact-mirror-image should be included in the installation object
 	cmd.Flags().StringVar(&localArtifactMirrorImage, "local-artifact-mirror-image", "", "Local artifact mirror image")
+	cmd.Flags().StringVar(&licenseID, "license-id", "", "License ID for online upgrades")
+	cmd.Flags().StringVar(&appSlug, "app-slug", "", "App slug for online upgrades")
+	cmd.Flags().StringVar(&channelSlug, "channel-slug", "", "Channel slug for online upgrades")
+	cmd.Flags().StringVar(&appVersion, "app-version", "", "App version for online upgrades")
```

Comment:
If we were using the cli binary rather than the operator im not sure any of these flags would be required and this would be more straightforward